### DISCUSSION TITLE
Selfhost

### DISCUSSION THREAD

**Discussion by classy-giraffe** [mm201/pkmnFoundations#89] [Q&A]
I'm having bad times reading this project code. I don't get what's the entrypoint. It lacks of documentation. It's pretty sad since i've enough hardware to selfhost a GTS for me and my friends. If somebody has managed to get this working, please reply to this issue.

**Comment by mm201** [comment]
This project is still very much a WIP and my current priorities are on getting things to work and making the site as good as possible.
It's a very straightforward ASP.NET project which should just work in Visual Studio. One way you can help is by improving documentation in the wiki.

**Comment by classy-giraffe** [comment]
Thanks for replying, I've just loaded the project inside Visual Studio Community 2019. It looks like the project miss a certain library called Gamestats (possibly a custom one made by you). Is that actually in the Repo?

**Comment by mm201** [comment]
Run `git submodule update --init`.
